executor/runtime/docker/docker_linux.go

```diff
@@ -254,9 +255,12 @@ func startSystemdUnit(ctx context.Context, conn *dbus.Conn, taskID string, cID s
 	select {
 	case <-ctx.Done():
 		doneErr := ctx.Err()
-
 		if doneErr == context.DeadlineExceeded {
-			return errors.Wrapf(doneErr, "timeout starting %s service", opts.humanName)
+			if opts.required {
+				return errors.Wrapf(doneErr, "timeout starting %s service", opts.humanName)
+			}
+			l.Errorf("timeout starting %s service (not required to launch this task)", opts.humanName)
```

Comment:
please use `withField`

Reply:
Will adding another field actually provide much value?
`l` already has the unitName (as well as taskid and containerid)

If we add the humanName too as a field, we'll have something like `ssh` (human name) *and* `titus-sshd` (unit name), I think I would rather have unitName consistently.